config/core/test-configs.yaml

```
@@ -1097,7 +1097,7 @@ device_types:
     boot_method: fvp
     filters:
       - passlist: {defconfig: ['defconfig']}
-
+      - regex: {'tree': '(next|mainline|stable|arm64)'}
```

Just to avoid matching other trees which contain these strings (e.g. `rt-stable`):
```suggestion
      - regex: {'tree': '^(next|mainline|stable|arm64)$'}
```